lib/segment.py

```diff
@@ -191,5 +191,7 @@ def projected_point(self, p):
 		p = np.array(p)
 		ap = p - a
 		ab = b - a
+		if ab[0] == 0 and ab[1] == 0:
+			return a
```

Comment:
I'm curious about this change.
Could you share an image that throws an error here, before the change?

Reply:
I'm not sure if i can post it here. Just sent U the sample image.

Reply:
Thanks for sending the image.
I can confirm that it was broken, and that your proposed fix works. :+1: